app/src/org/commcare/activities/CommCareWiFiDirectActivity.java

```diff
@@ -808,24 +809,7 @@ private void updateStatusText() {
     }
 
     public static void copyFile(InputStream inputStream, OutputStream out) throws IOException {
-        Logger.log(TAG, "File server copying file");
-        if (inputStream == null) {
-            Logger.log(TAG, "Input stream null");
-            throw new IOException("Got null input stream");
-        }
-        byte buf[] = new byte[1024];
-        int len;
-        try {
-            while ((len = inputStream.read(buf)) != -1) {
-                out.write(buf, 0, len);
-            }
-            out.close();
-            inputStream.close();
-        } catch (IOException e) {
-            Logger.exception("Copy in File Server failed with exception " + e, e);
-            throw e;
-        }
-        Logger.log(TAG, "Copy in File Server successful");
+        Log.d("SHIVAMMM", "copy file");

```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/b67bf14163879f9fb09208858b78318e65c45843/app/src/org/commcare/activities/CommCareWiFiDirectActivity.java#L812"></span>Using 'Log' instead of 'Timber'</td>
    </tr>
  </tbody>
</table>